content/600-about/100-prisma/40-faq.mdx

```diff
@@ -41,7 +41,7 @@ Prisma Client is not opinionated on how exactly you migrate your database schema
 
 GraphQL [schema delegation](https://www.prisma.io/blog/graphql-schema-stitching-explained-schema-delegation-4c6caf468405/) connects two GraphQL schemas by passing the [`info`](https://www.prisma.io/blog/graphql-server-basics-demystifying-the-info-argument-in-graphql-resolvers-6f26249f613a/) object from a resolver of the first GraphQL schema to a resolver of the second GraphQL schema. Schema delegation also is the foundation for [GraphQL binding](https://github.com/graphql-binding/graphql-binding).
 
-Prisma 1 officially supports both schema delegation and GraphQL binding as it exposes a GraphQL CRUD API through the [Prisma server](https://www.prisma.io/docs/prisma-server/). This API can be used to as the foundation for an application-layer GraphQL API created with GraphQL binding.
+Prisma 1 officially supports both schema delegation and GraphQL binding as it exposes a GraphQL CRUD API through the [Prisma server](https://v1.prisma.io/docs/1.34/prisma-server/). This API can be used to as the foundation for an application-layer GraphQL API created with GraphQL binding.
```

Comment:
Using the external URL also hid a link that was linking to an old URL that we are now redirecting somewhere else. 
Using the internal URL the link checker found this and failed with "broken link".

Reply:
@Jan, are the internal URL links not working? In the deploy version, they do... where are you seeing the link checker fail?

Reply:
In this case, `/prisma-server/` would have been the "internal" link. But that page does not exist, it is caught a few levels higher and redirected to the Prisma 1 docs. With the external link that was not caught by the link checker, as that only tests internal links. When I removed the domain, the link checker was triggered and failed because the page does not exist really.